homeassistant/components/stream/__init__.py

```diff
@@ -49,6 +54,18 @@ def request_stream(hass, stream_source, *, fmt="hls", keepalive=False, options=N
     if DOMAIN not in hass.config.components:
         raise HomeAssistantError("Stream integration is not set up.")
 
+    if DATA_UVLOOP_WARN not in hass.data:
+        hass.data[DATA_UVLOOP_WARN] = True
+        # Warn about https://github.com/home-assistant/home-assistant/issues/22999
+        if (
+            uvloop is not None
+            and isinstance(hass.loop, uvloop.Loop)
+            and "shell_command" in hass.config.components
+        ):
+            _LOGGER.warning(
+                "You are using UVLoop with stream and shell_command. This is known to cause issues. Please uninstall uvloop."
+            )
+
```

Comment:
It's not the only stream affected, also ffmpeg component

Reply:
Do you mean that we should add a warning to ffmpeg too? Or you mean stream + ffmpeg can crash too?

Reply:
First one, yes.

Reply:
ok, so if either shell_command OR ffmpeg loaded together with stream, warn